recipe/meta.yaml

```
@@ -24,16 +23,12 @@ requirements:
     - make
     - groff
   host:
-    - openssl 3.0.8
-    - cyrus-sasl 2.1.28
+    - openssl {{ openssl }}
+    - cyrus-sasl
```

https://git.openldap.org/openldap/openldap/-/blob/OPENLDAP_REL_ENG_2_6_10/README?ref_type=tags&plain=1#L17
```suggestion
    - cyrus-sasl 2.1.28
```